shared/js/dataDriver.js

```diff
@@ -85,15 +85,12 @@ loop.DataDriver = function() {
     }
 
     /**
-      * Add a page to the room's table of contents.
-      *
-      * @param {String} url       The url of the page.
-      * @param {String} title     The title of the page.
-      * @param {Object} metadata  Additional metadata,
-      *                           as provided by PageMetadata.
-      */
-    addPage(url, title, metadata) {
-      this.update("page", this.makeId(), { title, url, metadata });
+     * Add a page to the room's table of contents.
+     *
+     * @param {Object} page      The page object to add.
+     */
+    addPage(page) {
+      this.update("page", this.makeId(), { title: page.title, url: page.url });
```

Comment:
My original review comment was about the name `tile` and code positioning. I didn't say to split it into sub objects/fields:
```
This should be placed above sendTextChatMessage if we're going with the name add... And it should probably be named more explicitly to be a table of content tile as a generic tile could mean many things.
``